assets/src/js/batch.jsx

```diff
@@ -140,6 +142,11 @@ var App = React.createClass( {
 		this.toggleProcessing( true );
 	},
 
+	toggleResetModal : function ( active ) {
+		if ( active === false || active === true ) {
```

Comment:
Oh, good catch. Yea, we should probably keep things consistent. I'll add this to eslint too